coordinator/src/services/translation/request/send-message-payload.ts

```diff
@@ -96,3 +97,43 @@ function createControlActEventAuthor1(asid: string) {
   const agentSystemSds = new hl7V3.AgentSystemSds(agentSystemSystemSds)
   return new hl7V3.SendMessagePayloadAuthorSystemSds(agentSystemSds)
 }
+
+export function createReleaseRequestSendMessagePayload<T>(
```

Comment:
This is quite similar to createSendMessagePayload above. Could have a single method which accepts a controlActEvent to reduce duplication?